nextjs-app/components/pages/about/philosophy/ProgramIntroduction.tsx

```diff
@@ -0,0 +1,72 @@
+import Image from "next/image";
+import Routes from "@/constants/routes";
+import PageNavigationCard from "@/components/common/PageNavigationCard";
+
+const content: { title: string; description: string }[] = [
```

Comment:
```suggestion
const content = [
```

NIT: we don't need to specify the type, TS will do for us automatically